docs/source/release_notes/release-notes.rst

```diff
@@ -43,42 +43,45 @@ develop
 
-    *    - |improved|
-         - Add instrumentation to the thread pool used to run quorum checks during leader elections. This will be useful for
-           debugging `PaxosQuorumChecker can leave hanging threads <https://github.com/palantir/atlasdb/issues/1823>`.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1849>`__)
-
-    *    - |fixed|
-         - Fixed DbKvs.getRangeOfTimestamps() only returning the first page of results rather than paging through the whole range.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1872>`__)
-
-    *    - |fixed|
-         - Fixed a bug that would cause console to error on any range request that used a column selection and had more than one batch of results.
-           (`Pull Request <https://github.com/palantir/atlasdb/pull/1876>`__)
+    *    - |devbreak| |improved|
+         - The format of serialized exceptions occurring on a remote host has been brought in line with that of the `palantir/http-remoting <https://github.com/palantir/http-remoting>`__ library.
+           This should generally improve readability and also allows for more meaningful messages to be sent; we would previously return message bodies with no content for some exceptions (such as ``NotCurrentLeaderException``).
+           In particular, the assumption that a status code of 503 definitively means that the node being contacted is not the leader is no longer valid.
+           That said, existing AtlasDB clients will still behave correctly even with a new TimeLock.
+           (`Pull Request 1 <https://github.com/palantir/atlasdb/pull/1831>`__,
+           `Pull Request 2 <https://github.com/palantir/atlasdb/pull/1808>`__)
 
     *    - |new|
          - Timelock server now has jar publication in addition to dist publication.
            (`Pull Request <https://github.com/palantir/atlasdb/pull/1898>`__)
 
-    *    - |improved| |devbreak|
-         - The format of serialised exceptions occurring on a remote host has been brought in line with that of the `palantir/http-remoting <https://github.com/palantir/http-remoting>`__ library.
-           This should generally improve readability and also allows for more meaningful messages to be sent; we would previously return message bodies with no content for some exceptions (such as ``NotCurrentLeaderException``).
-           In particular, the assumption that a status code of 503 definitively means that the node being contacted the leader is no longer valid. That said, existing AtlasDB clients will still behave correctly even with a new TimeLock.
-           (`Pull Request 1 <https://github.com/palantir/atlasdb/pull/1831>`__,
-           `Pull Request 2 <https://github.com/palantir/atlasdb/pull/1808>`__)
-
     *    - |new| |fixed|
          - TimeLock clients may now receive a ``BlockingTimeoutException`` 503 if they make a lock request that blocks for longer than the server's idle timeout.
```

Comment:
Okay. I've clarified that this isn't just if you try to request for something more than the idle timeout, and also added that clients can retry (to emphasise that the request is actually valid):

> TimeLock clients may now receive an HTTP response with status code 503, encapsulating a ``BlockingTimeoutException``.
           This response is returned if a client makes a lock request that blocks for long enough that the server's idle timeout expires; clients may (immediately) retry the request.